lib/fishjam_web/controllers/healthcheck_controller.ex

```diff
@@ -23,14 +23,23 @@ defmodule FishjamWeb.HealthcheckController do
 
     conn
     |> put_resp_content_type("application/json")
-    |> render("show.json", report: report)
+    |> send_resp(200, Jason.encode!(%{data: report}))
   end
 
   defp get_health_report() do
     %{
-      status: :up,
+      localStatus: get_health_status(),
+      distributionEnabled: Application.fetch_env!(:fishjam, :dist_config)[:enabled],
+      nodesInCluster: length([Node.self() | Node.list()]),
+      nodesStatus: Fishjam.RPCClient.multicall(__MODULE__, :get_health_status, [])
```

Comment:
This line is problematic IMO. A healthcheck endpoint is expected to respond quickly (for instance, we use it to assess the liveness of Fishjam running in a container), here the multicall will slow it down significantly.

I'd suggest adding a separate endpoint for a cluster-wide healthcheck like this instead of extending this one.

Reply:
As long as the instances are in the same DC the time to make RPC is `negligible`.